### ISSUE TITLE
Dirty tab color icon gray again since many VS code versions

### ISSUE THREAD

**Issue by nicofurla** [wesbos/cobalt2-vscode#221]
Can't figure out if Im doing something wrong but dirty tab icon color is gray again (instead of yellow) since months now. 
Still using "Custom CSS and JS Loader" extensions (still necessary? There are no more hints about it in the readme file) and "cobalt2-custom-hacks.css" file, with latest edits (#154). Now way to get it yellow.

**Comment by wesbos** [comment]
Yeah the CSS hacks dont seem to work as well anymore. I wonder if I can do it without the CSS..

**Comment by wesbos** [comment]
Seems the CSS changed, you can use this:

```css
[role="tab"].dirty a.codicon-close:before {
  color: #ffc600;
}
```

**Comment by nicofurla** [comment]
Thank you, it worked!
Can you please tell where is this CSS updated? As told, there's no more mention of CSS and extension in the readme, hence I thought they weren't necessary anymore; and didn't find anything in issues/ pull requests too. Maybe on a fork?
And, secondly... yes, it would be great if all of that styling could be done without the CSS.

**Comment by wesbos** [comment]
https://github.com/wesbos/cobalt2-vscode/tree/a40d2011cb601bfc9ae5ba153d38c86b95356a9d#important-css-hacks-for-extra-sweet-editor
